Packs/MISP/Integrations/MISPV3/MISPV3.yml

```diff
@@ -2819,6 +2819,27 @@ script:
     description: Delete an attribute according to the given attribute ID.
     execution: false
     name: misp-delete-attribute
+  - arguments:
+    - default: false
+      description: Event ID to be published.
+      isArray: false
+      name: event_id
+      required: true
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Whether to send an email. The default is to not send a mail.
```

Comment:
Why the mismatch between "alert" and "email"?

Reply:
I wanted to follow PYMISP args conventions (this arg called their 'alert'), I can change it if you prefer.